### PR TITLE
Projects: Remove /projects from internal links

### PR DESCRIPTION
**Pull request by eatyourgreens**
#2519 added `/projects` as a base path for the project app. This PR updates links, which have been hardcoded with `/projects` and now point to `/projects/projects/:owner:/:name:`.

Package:
app-project

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/app-project/src/screens/ProjectAboutPage/components/TeamMember/TeamMember.spec.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ describe('Component > TeamMember', function () {
     const link = getByRole('link')
     expect(link).exists()
     expect(link.href).include(
-      '/projects/zooniverse/snapshot-serengeti/users/mock_user'
+      '/zooniverse/snapshot-serengeti/users/mock_user'
```
Comment (camallen) [inline packages/app-project/src/screens/ProjectAboutPage/components/TeamMember/TeamMember.spec.js:24]: Wouldn't this link href still include the projects path or are these relative links?

**Review by camallen — APPROVED** [review]
LGTM - assuming tests pass etc. 

Minor question about link testing and if we use relative vs absolute links, not blocking.

File: packages/app-project/src/screens/ProjectAboutPage/components/TeamMember/TeamMember.spec.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/app-project/src/screens/ProjectAboutPage/components/TeamMember/TeamMember.spec.js:24]: I think these are using the `next/link` component, which dynamically decides whether or not to use a client-side link and prefixes it with `basePath` for you. I'll double-check, but I'm pretty sure this change fixes the links on the Team page.

Comment (eatyourgreens) [reply packages/app-project/src/screens/ProjectAboutPage/components/TeamMember/TeamMember.spec.js:24]: Double checked, now that deploys are working again, and this change is needed to fix links to team members' profiles on the Team page.

**Pull request merged**